tests/mux/test_mux.sh

```diff
@@ -90,7 +90,7 @@ printf "Testing server disconnection death... "
 sleep 1 && kill `cat $SOCKK.pid`
 rm $SOCKK $SOCKK.pid
 sleep 1
-if pgrep -F $SOCKM.pid | grep .; then
+if kill -s -0 $SOCKM.pid 2> /dev/null; then
```

Comment:
Should this be `kill -s -0` or `kill -s 0`?

Reply:
... and is there supposed to be a `cat $SOCKM.pid` here?

Reply:
You're correct on both counts.

Updated with a rebased branch.